cognite/neat/utils/utils.py

```diff
@@ -192,7 +220,9 @@ def _traverse(hierarchy: dict, graph: dict, names: list[str]) -> dict:
 
 
 def get_generation_order(
-    class_linkage: pd.DataFrame, parent_col: str = "source_class", child_col: str = "target_class"
+    class_linkage: pd.DataFrame,
+    parent_col: str = "source_class",
+    child_col: str = "target_class",
```

Comment:
Is this a preference? Or do you have some sort of plugin that changes the style?

Not a big deal, but all these irrelevant multiline changes makes the PR a bit more difficult to review.